File: C/alphabet_counter.c
#include <stdio.h>
void main() {
    int i = 0; int j = 0; int count = 0;
    char string[100];
    printf("Enter a string.\n");
    fgets(string, 100, stdin);
    while(string[i] != '\0'){
        j = 0;
        count = 0;
        while(string[j] != '\0'){
            if(string[i] == string[j]){
                count++;
            }
            j++;
        }
        printf("The count of %c is %d\n", string[i], count);
        i++;
    }
}

File: Python/alphabet counter/alphabetCounter.py
input_string = input("Please enter a string.\n")
string_length = len(input_string)

i = 0
while i < string_length:
    j = 0
    count = 0
    while j < string_length:
        if input_string[j] == input_string[i]:
            count = count + 1
        j = j+1
    print("The count of {} is {}".format(input_string[i],count))
    i = i+1
    